### ISSUE TITLE
Plenty of error message "PlainTextBytes: Not found" in log/gitea.log

### ISSUE THREAD

**Issue by lihanglin** [go-gitea/gitea#20137]
### Description

When examining my log/gitea.log, there are plenty of the error messages as below.

```
2022/06/26 10:10:40 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:10:42 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:10:44 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:10:59 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:10:59 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:11:30 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:11:30 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
2022/06/26 10:11:36 ...s/context/context.go:314:PlainTextBytes() [E] PlainTextBytes: Not found.
```

It shows up every 3~5 seconds but the gitea service works.
I am curious if there are any configuration I set it wrong?
This message is flooding my log file.

### Gitea Version

1.16.8

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

https://gist.github.com/lihanglin/c6f6c5bc070d938283a428f2f3db8960

### Git Version

2.34.1

### Operating System

CentOS Linux release 7.9.2009

### How are you running Gitea?

I use the official release gitea and executed it native on host.

### Database

MySQL

**Comment by wxiaoguang** [comment]
It's likely that because you changed the default log level, then you can not see the router/access logs, only see the `Not Found` error in logs. There must be someone is accessing non-existing URL.

Set your log level to `info` or `debug` for more details.

**Comment by zeripath** [comment]
The logging of ` PlainTextBytes: Not found` was excessive and we have removed this in #18421 which will be in 1.17

Fixed by #18421
